ts/features/lollipop/types/LollipopLoginSource.ts

```diff
@@ -0,0 +1,28 @@
+type LoginSource = {
```

Comment:
This type is a redefinition of `WebViewSourceUri` (`import { WebViewSourceUri } from "react-native-webview/lib/WebViewTypes";`)

Can we avoid to redefined it and use `WebViewSourceUri` directly under the type `LoginSourceReady`?